docs/pages/virtual.md

```diff
@@ -5,10 +5,21 @@ layout: default
 
 # Virtual environments
 
-| Name                                                     | Short description                                                         | 🚦  |
-| -------------------------------------------------------- | ------------------------------------------------------------------------- | :-: |
-| [Conda](https://docs.conda.io/projects/conda/en/stable/) | Installs, runs, and updates packages and their dependencies.              | 🟢  |
-| [pipenv](https://pipenv.pypa.io/en/latest/)              | Automatically creates and manages a virtualenv for your projects.         | 🟠  |
-| [pyenv](https://github.com/pyenv/pyenv)                  | Lets you easily switch between multiple versions of Python.               | 🟠  |
-| [virtualenv](https://virtualenv.pypa.io/en/latest/)      | Creates isolated Python environments, and offers more features than venv. | 🟠  |
-| [venv](https://docs.python.org/3/library/venv.html)      | Creates isolated Python environments.                                     | 🔴  |
+| Name                    | Short description                                                                                                                                                                         | 🚦  |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-: |
+| conda-forge [miniforge] | Installs, runs, and updates packages and their dependencies. A trusted sub-set of `conda` with the `conda-forge` channel.                                                                 | 🟢  |
+| [pipenv]                | Automatically creates and manages a virtualenv for your projects.                                                                                                                         | 🟠  |
+| [pyenv]                 | Lets you easily switch between multiple versions of Python.                                                                                                                               | 🟠  |
+| [virtualenv]            | Creates isolated Python environments, and offers more features than venv.                                                                                                                 | 🟠  |
+| [anaconda]              | Due to recent [licensing ambiguity][anaconda-problems], we recommend avoiding anaconda and many of the default channels. We recommend miniforge or sticking to the `conda-forge` channel. | 🔴  |
+| [venv]                  | Creates isolated Python environments.                                                                                                                                                     | 🔴  |
+
+<!-- links here for a more readable table -->
+
+[miniforge]: https://github.com/conda-forge/miniforge
```

Comment:
```suggestion
[miniforge]: https://conda-forge.org/download/
```
Is maybe a slightly more user friendly link that gets you to an installer quicker?